postman/schemas/transfer.yml

```diff
@@ -11,6 +11,665 @@ servers:
 security:
 - bearerAuth: []
 paths:
+  "/account-fields":
+    get:
+      tags:
+        - Accounts
+      summary: Get Account Fields
+      description: Get Account Fields
```

Comment:
Missing a description. What is this endpoint used for? How is it different than GET /recipient-fields?